### PR TITLE
Fix some typos.

### PR DESCRIPTION
**Pull request by nathggns**
Just fixes some typos in the spriting documentation.

### CODE REVIEW

File: doc-src/content/help/tutorials/spriting.markdown [hunk 1]
```diff
@@ -52,14 +52,14 @@ You can now apply the `icon-XXX` classes to your markup as needed.
 
 Let's go over what happened there. The import statement told compass to [generate a
 stylesheet that is customized for your sprites](https://gist.github.com/729507). This
-stylesheet is [magic](/help/tutorials/spriting/magic-imports), it is not written to disk, and it can be customized
+stylesheet is [magic](#magic-imports), it is not written to disk, and it can be customized
 by setting configuration variables before you import it. See the section below on
-[Customization Options](/help/tutorials/spriting/customization). The goal of this stylesheet is to provide a
```
Comment (scottdavis) [inline doc-src/content/help/tutorials/spriting.markdown:57]: This should not be changed this way the correct way would be to change the link to "/help/tutorials/spriting/customization-options"

Comment (nathggns) [inline doc-src/content/help/tutorials/spriting.markdown:57]: Well, no. The forward slashes are creating the problems, they're supposed to be hashes. Or do you mean the names?

Comment (scottdavis) [inline doc-src/content/help/tutorials/spriting.markdown:57]: i had typo the url to the link the [Customization Options] should link to the other page

Comment (scottdavis) [inline doc-src/content/help/tutorials/spriting.markdown:57]: http://beta.compass-style.org/help/tutorials/spriting/customization-options/

**Comment by nathggns** [comment]
Messed up the repo some how. Will make a new pull request...

**Pull request closed without merging**